The value of `minimumReleaseAge` is set too high

During the last few days, I keep running into issues where I can't install packages or run commands because of the high value in `minimumReleaseAge`.


I think a week is way too long. 

We try to avoid installing packages that got hijacked but in most cases, these things are typically address within hours, not days.

It hurts the DX and to get around it - make sure I can move on with the thing I try to do - I go into the file, lower the value and set it back.

I think we should move it back to 1 day.

## Example

```
$ pnpm dlx shadcn@latest add pagination
✔ Checking registry.
⠋ Installing components.

Something went wrong. Please check the error below for more details.
If the problem persists, please open an issue on GitHub.

Command failed with exit code 1: pnpm add '@radix-ui/react-slot'

../..                                    | Progress: resolved 1, reused 0, downloaded 0, added 0
/Users/beeman/dev/samui-build/samui-wallet:
\u2009ERR_PNPM_NO_MATCHING_VERSION\u2009 No matching version found for vitest@4.0.4 published by Mon Oct 27 2025 13:57:36 GMT+0400 (Gulf Standard Time) while fetching it from https://registry.npmjs.org/. Version 4.0.4 satisfies the specs but was released at Mon Oct 27 2025 16:29:23 GMT+0400 (Gulf Standard Time)

This error happened while installing a direct dependency of /Users/beeman/dev/samui-build/samui-wallet

The latest release of vitest is "4.0.6". Published at 10/31/2025

Other releases are:
  * beta: 4.0.0-beta.19 published at 10/21/2025

If you need the full list of all 438 published versions run "$ pnpm view vitest versions".

If you want to install the matched version ignoring the time it was published, you can add the package name to the minimumReleaseAgeExclude setting. Read more about it: https://pnpm.io/settings#minimumreleaseageexclude

```

--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -30,6 +30,6 @@ catalog:
   ws: 8.18.3
   zod: ^4.1.12
 
-minimumReleaseAge: 10080
+minimumReleaseAge: 1440
 
 engineStrict: true

